examples/features/mutually-awared-nses/README.md

```diff
@@ -0,0 +1,178 @@
+# Test kernel to kernel connection
+
+
+This example shows that NSC and NSE on the one node can find each other. 
+
+NSC and NSE are using the `kernel` mechanism to connect to its local forwarder.
+
+## Requires
+
+Make sure that you have completed steps from [basic](../../basic) or [memory](../../memory) setup.
+
+## Run
+
+Create test namespace:
+```bash
+NAMESPACE=($(kubectl create -f https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/ff873ab1eec320c2abf6ba380d95137e71a969b2/examples/use-cases/namespace.yaml)[0])
+NAMESPACE=${NAMESPACE:10}
+```
+
+Select node to deploy NSC and NSE:
+```bash
+NODE=($(kubectl get nodes -o go-template='{{range .items}}{{ if not .spec.taints  }}{{index .metadata.labels "kubernetes.io/hostname"}} {{end}}{{end}}')[0])
+```
+
+Create customization file:
+```bash
+cat > kustomization.yaml <<EOF
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: ${NAMESPACE}
+
+resources:
+- nse-1.yaml
+- nse-2.yaml
+- config-file-nse-1.yaml
+- config-file-nse-2.yaml
+bases:
+- https://github.com/networkservicemesh/deployments-k8s/apps/nsc-kernel?ref=ff873ab1eec320c2abf6ba380d95137e71a969b2
+
+patchesStrategicMerge:
+- patch-nse-1.yaml
+- patch-nse-2.yaml
```

Comment:
Do we really need these patch?

Also can we replace nodes to **Affinity and anti-affinity**
https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity?